Pipelines build task Android Signing Fix explanation for keystore file (#3706)
MicrosoftDocs/vsts-docs: docs/pipelines/tasks/build/android-signing.md

<tr>
<td>Keystore File</td>
<td>

> Enter the secure file ID of the keystore file that should be used to sign the APK. To save the keystore file, go to Piplines > Library > Secure files and upload the secure file there. To find the secure file ID, click on the keystore file you have uploaded in the list of secure file and, in the page URI, copy the secureFileId property.

Reviewer: ```suggestion
Enter the secure file name of the keystore file that should be used to sign the APK. To save the keystore file as a secure file, go to Piplines > Library > Secure files and upload it.
```

Author: I agree, it makes more sense to use the file name instead of the ID, it's great to know you can do that now😄.
